### DISCUSSION TITLE
Running Sailor Hat WiFi Gateway on SH-ESP32

### DISCUSSION THREAD

**Discussion by nadrog** [hatlabs/discussions#37] [Q&A]
Hi,

  is it possible to run SH-wg software on SH-ESP32 hardware?
  If yes, what hardware is assumed by SH-wg and is missing on SH-ESP32?

Thanks, Gordan

**Comment by mairas** [comment]
Unfortunately, that's not a supported configuration. If you want to use and/or modify the SH-wg-firmware, I recommend purchasing a [SH-wg](https://shop.hatlabs.fi/products/sh-wg) device to do that.
